src/resolveFormatter.test.ts

```diff
@@ -102,4 +102,39 @@ describe("resolveFormatter", () => {
 
 		expect(formatter).toBeUndefined();
 	});
+
+	it("resolves with a default order", async () => {
```

Comment:
[Naming] Nit, whenever I find myself wanting to comment in a test it feels like a sign to make the test title more descriptive:

```suggestion
	it("prefers Biome over Prettier when no order is given, Biome's config file exists, and Prettier's config file exists", async () => {
```

(this will be more useful if&when >=1 test gets added to fix the strategy-ordering bug)